src/main/scala/com/microsoft/hyperspace/index/IndexCollectionManager.scala

```diff
@@ -34,11 +34,13 @@ class IndexCollectionManager(
   private val conf: SQLConf = spark.sessionState.conf
 
   override def create(df: DataFrame, indexConfig: IndexConfig): Unit = {
-    val indexPath = PathResolver(spark.sessionState.conf).getIndexPath(indexConfig.indexName)
-    val dataManager = indexDataManagerFactory.create(indexPath)
+    val indexPath = PathResolver(spark.sessionState.conf,
+      spark.sessionState.newHadoopConf()).getIndexPath(indexConfig.indexName)
+    val dataManager = indexDataManagerFactory.create(indexPath,
+      spark.sessionState.newHadoopConf())
```

Comment:
Can you run scalafmt on this file? I see a different formatting:
```scala
    val indexPath = PathResolver(spark.sessionState.conf, spark.sessionState.newHadoopConf())
      .getIndexPath(indexConfig.indexName)
    val dataManager =
      indexDataManagerFactory.create(indexPath, spark.sessionState.newHadoopConf())
```

Reply:
👍